manifests/templates/file-host.yaml.in

```diff
@@ -48,6 +60,18 @@ spec:
         volumeMounts:
         - name: "images"
           mountPath: "/tmp/shared"
+        readinessProbe:
+          httpGet:
+            path: /minio/health/ready
```

Comment:
is this path provided by minio?

Reply:
Yes - we're using an official minio container which provides this for kubernetes.